Makefile

```diff
@@ -2,10 +2,12 @@ isort:
 	isort -rc -vb .
 
 test:
-	py.test --isort
+	if [ "$$TRAVIS_PYTHON_VERSION" != "3.5" ]; then \
+	py.test --isort --ignore=tests/asyncio; \
+	else py.test --isort; fi
```

Comment:
@ticosax 
Could this make use of https://pypi.python.org/pypi/pytest-asyncio instead?